ext/standard/tests/general_functions/dl-001.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+dl(): Loaded extensions properly unregister their ini settings
+--SKIPIF--
+<?php include __DIR__ . "/../../../dl_test/tests/skip.inc";
```

Comment:
I very much prefer to explicitly have the closing PHP tags in .phpt files, so that syntax highlighting works.

Reply:
No. The sections containing PHP code are extracted to temporary files, so closing tags are not strictly needed.

Reply:
For me personally it's because I like to be able to run tests quickly without invoking run-tests, by just doing `php thetest.phpt`